app/frontend/Calendar/CalendarApp.svelte

```diff
@@ -20,7 +20,7 @@
   import { mergeColls } from "svelte-collections";
   import { t } from "../../l10n/l10n";
 
-  $: events = mergeColls(appGlobal.calendars.map(cal => cal.fillRecurrences(new Date(Date.now() + 1e11)))).sortBy(ev => ev.startTime);
+  $: events = mergeColls(appGlobal.calendars.map(cal => cal.fillRecurrences(new Date(Date.now() + 1e11)))).filter(ev => !ev.recurrenceRule).sortBy(ev => ev.startTime);
```

Comment:
(Not about your current change, but about the `fillRecurrences()` etc.)
Are you aware that this runs every time we open the calendar view?
Could we cache this somehow?

Reply:
We already cache the results; we only run this to ensure that the cache is up-to-date.